middle_end/flambda/lift_let_to_initialize_symbol.ml

```diff
@@ -112,7 +181,8 @@ let rec accumulate ~(layouts : Layouts.t) ~substitution ~copied_lets ~extracted_
           Flambda_utils.toplevel_substitution substitution
             (Flambda.create_let renamed named (Var renamed))
         in
-        Expr (var, [Field 0], expr)
+        let expr, additionnal_path = pack_expr ~layouts expr in
```

Comment:
Maybe add a comment in the `Let_rec` cases that since `layout_letrec` ensures we have values, we don't need to call `pack_expr`? (so that we remember to change this in the future if it letrecs are no longer necessarily values)